### PR TITLE
Add more logical record types to Python

### PR DESCRIPTION
**Pull request by ErlendHaa**

### CODE REVIEW

File: python/dlisio/plumbing/longname.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+        #: Conditions applicable at the time the information was acquired or
+        #: generated
+        self.conditions      = []
+
+        #: Industry-standardized symbolic name by which the information is
+        #: known. The possible values are specified by POSC
+        self.standard_symbol = None
+
+        #:
```
Comment (achaikou) [inline python/dlisio/plumbing/longname.py]: Maybe "Association between the recorded information and corresponding records or objects of the Producer’s internal or corporate database"?

File: python/tests/test_semantics.py [hunk 2]
```diff
@@ -75,6 +75,34 @@ def test_origin(f):
     assert random_origin.file_set_nr == 1042
     assert random_origin.file_nr     == 6
 
+def test_axis(f):
```
Comment (achaikou) [inline python/tests/test_semantics.py]: Btw, I still believe that core tests on actual files might be beneficial. Maybe there is no need to test every single attribute, but checking number of objects or whatever might still provide some value.

Comment (ErlendHaa) [reply python/tests/test_semantics.py]: I have looked through all of the test files, and none of them contains objects such as long_name, axis or zone! Maybe we can get some additional testfiles in the future

File: python/tests/test_semantics.py [hunk 3]
```diff
@@ -75,6 +75,34 @@ def test_origin(f):
...
+    assert ln.entity_nr       == '21'
+    assert ln.entity_part     == 'top part?'
+    assert ln.entity_part_nr  == '1'
+    assert ln.generic_source  == 'random words'
+    assert ln.source_part     == ['generator']
+    assert ln.source_part_nr  == ['8412']
+    assert ln.conditions      == ['at standard temperature']
+    assert ln.standard_symbol == 'SYM'
+    assert ln.private_symbol  == 'BOL'
+
```
[the remark this answers is not in the document]
Comment (ErlendHaa) [reply python/tests/test_semantics.py]: Agree! We need to update all links to these new objects from the existing ones, but it makes sense to wait for the new link-routine.

[the remark this answers is not in the document]
Comment (ErlendHaa) [reply python/tests/test_semantics.py]: if #108 is merged first, I will update linking for these and existing objects! With tests ofc :D

File: python/dlisio/plumbing/computation.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        #: computation is constant
+        self.zones       = []
+
+        #: Computational values
+        self.values      = []
+
+        #: The immediate source of the Computation
+        self.source = []
+
+        #: Reference to the source object
```
Comment (achaikou) [inline python/dlisio/plumbing/computation.py]: You shouldn't need it

**Pull request merged**